### PR TITLE
Add global options to poll create

### LINKED ISSUE #1939: poll create action misses amount_global_* options
In the action wiki are the three optional options:
{{{
    amount_global_yes?: decimal(6);
    amount_global_no?: decimal(6);
    amount_global_abstain?: decimal(6);
}}}
They are missing in the implementation of `poll.create`.

This is needed for https://github.com/OpenSlides/openslides-client/issues/2583

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #1939

### CODE REVIEW

File: tests/system/action/poll/test_create.py [hunk 1]
```diff
@@ -62,10 +63,13 @@ def test_create_correct(self) -> None:
         assert option.get("text") == "test2"
         assert option.get("poll_id") == 1
         assert option.get("meeting_id") == 1
+        assert option.get("yes") == "10.000000"
         global_option = self.get_model("option/2")
```
Comment (r-peschke) [inline tests/system/action/poll/test_create.py:67]: A hint, you don't need to change it here:
We have a `self.assert_model_exists` - method for reading and asserting the values.

**Review by r-peschke — APPROVED** [review]
Just one small thing and one hint. After changing you can merge directly

File: tests/system/action/poll/test_create.py (hunk 1 above)
Comment (luisa-beerboom) [reply tests/system/action/poll/test_create.py]: Done

File: openslides_backend/action/actions/poll/create.py [hunk 2]
```diff
@@ -156,6 +159,12 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             "meeting_id": instance["meeting_id"],
             "weight": 1,
         }
+        for option in ["yes", "no", "abstain"]:
+            if instance["type"] == "analog" and instance.get(f"global_{option}"):
```
[the remark this answers is not in the document]
Comment (luisa-beerboom) [reply openslides_backend/action/actions/poll/create.py]: Done

**Pull request merged**